app/src/org/commcare/engine/extensions/IntentExtensionParser.java

```diff
@@ -42,7 +42,8 @@ public void handle(XFormParser p, Element e, Object parent) {
 
         Log.d(TAG, "0123 extention parser appearance is: " + appearance);
 
-        String label = e.getAttributeValue(null, "button-label");
+        String getButtonLabel = e.getAttributeValue(null, "button-label");
```

Comment:
variable name here could be clearer, not sure what the "get" part is meant to mean